pkg/k8s/apis/cilium.io/client/crds/v2/ciliumclusterwidenetworkpolicies.yaml

```diff
@@ -548,15 +548,15 @@ spec:
                               description: PortProtocol specifies an L4 port with
                                 an optional transport protocol
                               properties:
+                                endPort:
+                                  description: EndPort can be an L4 port number.
```

Comment:
Can it be something else like the port definition:`or a name in the form of "http" or "http-8080`?